src/Sylius/Bundle/ChannelBundle/Doctrine/ORM/ChannelRepository.php

```diff
@@ -24,28 +22,8 @@ class ChannelRepository extends EntityRepository implements ChannelRepositoryInt
     /**
      * {@inheritdoc}
      */
-    public function findMatchingHostname($hostname)
-    {
-        $queryBuilder = $this->getQueryBuilder();
-
-        $queryBuilder
-            ->andWhere($queryBuilder->expr()->like('o.url', ':hostname'))
-            ->setParameter('hostname', '%'.$hostname.'%')
-        ;
-
-        return $queryBuilder->getQuery()->getOneOrNullResult();
-    }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function findDefault()
```

Comment:
It would be good to support single channel without need to configure hostname or anything else, maybe we should have a default channel? We could have the default channel resolver that will be the last resolver and will return the channel that is marked as default.